Yank/tests/test_yank.py

```diff
@@ -70,15 +70,38 @@ def test_topography():
     assert len(topography.ions_atoms) == 0
 
 
+def test_topography_regions():
+    """Test that topography regions are created and fetched"""
+    toluene_vacuum = testsystems.TolueneVacuum()
+    topography = Topography(toluene_vacuum.topology)
+    # Should do nothing and return nothing without error
+    assert topography.remove_region('Nothing') is None
```

Comment:
I'm not sure if this is what you wanted, but currently, the `remove_region` method doesn't return anything, so the test would always pass.

Reply:
It is what I wanted. I wanted the test that trying to remove a non-existing region does nothing and does not return an error. The `assert` ensures we in fact are getting the correct output and the call itself ensures this corner case is tested